tools/tcpconnect.py

```diff
@@ -107,6 +111,8 @@
     u16 dport = 0;
     bpf_probe_read(&dport, sizeof(dport), &skp->__sk_common.skc_dport);
 
+    FILTER_PORT
```

Comment:
@brendangregg I have tried to put this in `trace_connect_entry` but did not manage do it there. I can try to dig deeper if needed.